codefresh/docker/release.yaml

```
@@ -0,0 +1,34 @@
+version: '1.0'
+
+stages:
+  - Prepare
+  - Promote
+
+steps:
+  main_clone:
+    title: "Create Context"
+    stage: "Prepare"
+    image: alpine
+    commands:
+      # Extract the postfix of a semver (e.g. 0.0.0-stage => stage)
+      - cf_export STAGE=$(echo ${{CF_RELEASE_TAG}} | sed -E 's/^[^-]+-?//')
+
+  pull_image_sha:
+    title: Pull image with commit sha
+    stage: "Promote"
+    image: "${{CF_DOCKER_REPO_URL}}/${{CF_REPO_NAME}}:${{CF_REVISION}}"
+    retry:
+      maxAttempts: 40
```

```suggestion
      maxAttempts: 10
```

Does Codefresh have max attempts 10 with max delay 60?